torch/csrc/autograd/autocast_mode.h

```diff
@@ -0,0 +1,14 @@
+#pragma once
+
+#include <ATen/core/autocast/autocast_mode.h>
+#include <torch/csrc/WindowsTorchApiMacro.h>
+
+// I'm not sure why this header is even necessary (couldn't I just say
+// at::AutocastMode directly in init.cpp??) but I'm imitating grad_mode.h,
+// which presumably had good intentions.
```

Comment:
Probably not, actually. More likely we were moving files around and kept old files around for BC.

Reply:
Removed the header, now referencing at::AutocastMode directly in the autograd/init.cpp functions.